config/locales/fr.yml

```
@@ -40,7 +40,8 @@ fr:
     i_dont_know: Je ne sais pas
     deconnexion: "Déconnexion"
     pj: "Pièces jointes"
-    asterisk_html: Les champs suivis d’un astérisque ( <span class = mandatory> * </span> ) sont obligatoires.
+    asterisk_html: Les champs suivis d’un astérisque ( <span class> * </span> ) sont obligatoires.
```

```suggestion
    asterisk_html: Les champs suivis d’un astérisque ( * ) sont obligatoires.
```
(à faire aussi dans le en.yml)

 ça m'ennuie un peu  de ne pas réutiliser le même svg ici (suivant les navigateurs elle paraît très différente), mais j'ai pas d'idée pour le moment de comment faire ça